src/sagemaker/utils.py

```diff
@@ -122,6 +122,10 @@ def get_config_value(key_path, config):
     return current_section
 
 
+def get_short_version(framework_version):
+    return '.'.join(framework_version.split('.')[:2])
```

Comment:
I wonder if this would be cleaner with a regex. I guess with such a short string it doesn't really matter.

Reply:
yeah. leaving as is if you don't feel strongly about it.